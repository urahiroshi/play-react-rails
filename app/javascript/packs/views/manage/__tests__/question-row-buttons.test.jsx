import React from 'react'
import renderer from 'react-test-renderer'
import QuestionRowButtons from '../question-row-buttons'

describe('#render', () => {
  it('should show snapshot', () => {
    const component = renderer.create(
      <QuestionRowButtons
        onClickEdit={() => {}}
        onClickDelete={() => {}}
      />,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
