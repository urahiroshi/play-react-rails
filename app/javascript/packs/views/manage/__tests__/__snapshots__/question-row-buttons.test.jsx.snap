// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<div>
  <button
    className="btn btn-warning"
    onClick={[Function]}
  >
    Edit
  </button>
  <button
    className="btn btn-danger"
    onClick={[Function]}
  >
    Delete
  </button>
</div>
`;
